### PR TITLE
Revert items changes prepare4.1.0

### PR DESCRIPTION
**Pull request by leerho**
Removed partial ItemsSketch work.

Left some TODOs as place holders.

Left improvements over 4.0.0.

This becomes the basis for 4.1.0 changes.

Runs all unit tests.

### CODE REVIEW

**Comment by leerho** [comment]
This is the basis for 4.1.0.

File: src/main/java/org/apache/datasketches/kll/KllHelper.java [hunk 1]
```diff
@@ -237,10 +244,11 @@ static void compressWhileUpdatingSketch(final KllSketch sketch) {
       }
       for (int lvl = 0; lvl < level; lvl++) {
         newIndex = myLevelsArr[lvl] + halfAdjPop; //adjust boundary
-        sketch.setLevelsArrayAt(lvl, newIndex);
+        fltSk.setLevelsArrayAt(lvl, newIndex);
       }
-      sketch.setFloatItemsArray(myFloatItemsArr);
+      fltSk.setFloatItemsArray(myFloatItemsArr);
     }
+    //TODO add items
```
Comment (AlexanderSaydakov) [inline src/main/java/org/apache/datasketches/kll/KllHelper.java:251]: what does this mean?

File: src/main/java/org/apache/datasketches/kll/KllHelper.java [hunk 2]
```diff
@@ -31,19 +31,22 @@
...
 import static org.apache.datasketches.kll.KllPreambleUtil.PREAMBLE_INTS_EMPTY_SINGLE;
 import static org.apache.datasketches.kll.KllPreambleUtil.PREAMBLE_INTS_FULL;
 import static org.apache.datasketches.kll.KllPreambleUtil.SERIAL_VERSION_EMPTY_FULL;
 import static org.apache.datasketches.kll.KllPreambleUtil.SERIAL_VERSION_SINGLE;
 import static org.apache.datasketches.kll.KllPreambleUtil.SERIAL_VERSION_UPDATABLE;
+import static org.apache.datasketches.kll.KllPreambleUtil.SER_VER_BYTE_ADR;
 import static org.apache.datasketches.kll.KllPreambleUtil.SINGLE_ITEM_BIT_MASK;
 import static org.apache.datasketches.kll.KllPreambleUtil.UPDATABLE_BIT_MASK;
-import static org.apache.datasketches.kll.KllPreambleUtil.setMemoryDoubleSketchFlag;
+//import static org.apache.datasketches.kll.KllPreambleUtil.setMemoryDoubleSketchFlag;
```
Comment (AlexanderSaydakov) [inline src/main/java/org/apache/datasketches/kll/KllHelper.java:49]: should this be removed?

File: src/main/java/org/apache/datasketches/kll/KllHelper.java [hunk 3]
```diff
@@ -347,7 +355,7 @@ static GrowthStats getGrowthSchemeForGivenN(
       println("Given N         : " + gStats.givenN);
       printf("%10s %10s %20s %13s %15s\n", "NumLevels", "MaxItems", "MaxN", "CompactBytes", "UpdatableBytes");
     }
-    final int typeBytes = (sketchType == DOUBLES_SKETCH) ? Double.BYTES : Float.BYTES;
+    final int typeBytes = sketchType == DOUBLES_SKETCH ? Double.BYTES : Float.BYTES;
     do {
       gStats.numLevels++; //
```
Comment (AlexanderSaydakov) [inline src/main/java/org/apache/datasketches/kll/KllHelper.java:360]: empty comment?

File: src/main/java/org/apache/datasketches/kll/KllHelper.java (hunk 2 above)
Comment (leerho) [reply src/main/java/org/apache/datasketches/kll/KllHelper.java:49]: Done.

File: src/main/java/org/apache/datasketches/kll/KllHelper.java (hunk 3 above)
Comment (leerho) [reply src/main/java/org/apache/datasketches/kll/KllHelper.java:360]: Done.

**Comment by leerho** [comment]
You must have removed this because it doesn't show up on github

**Comment by AlexanderSaydakov** [comment]
I tried reading and writing KLL float and double sketches from C++ in Java and the other way around. This seems to work fine.

**Comment by AlexanderSaydakov** [comment]
I don't see why some package private methods are needed such as getMinFloatItem and getMinDoubleItem

**Review by AlexanderSaydakov — APPROVED** [review]

**Pull request merged**